icons.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+install -dm755 /app/share/icons/hicolor/32x32/status
+install -dm755 /app/share/icons/hicolor/scalable/status
+
+# Indicator 32px #
+
+cp /app/share/icons/hicolor/32x32/status/laptopconnected.png /app/share/icons/hicolor/32x32/status/org.kde.kdeconnect.laptopconnected.png
```

Comment:
Can you elaborate on why this is needed?

Reply:
the icons don't load here if it is not exported, and flatpak only exports icons with the app-id